modules/sfm/CMakeLists.txt

```diff
@@ -0,0 +1,122 @@
+set(the_description "SFM algorithms")
+
+
+### LIBMV LIGHT EXTERNAL DEPENDENCIES ###
+
+find_package(Ceres QUIET)
+
+if(NOT DEFINED SFM_DEPS_OK)
+  set(_fname "${CMAKE_CURRENT_BINARY_DIR}/test_sfm_deps.cpp")
```

Comment:
Please add this code to resolve issue with ARM compiler (cross-compiler is buggy and has access to host include files in /usr/include , so we force check for glog library here):
```
  if(NOT DEFINED GLOG_LIBRARIES)
    set(GLOG_LIBRARIES "glog")
  endif()
```